include/tango/server/command.h

```diff
@@ -220,9 +222,10 @@ class Command
  * The object desctructor.
  */
 #ifdef HAS_UNIQUE_PTR
-    virtual ~Command() {}
+    virtual ~Command()
+    {}
 #else
-	virtual ~Command() {delete ext;}
+    virtual ~Command() {delete ext;}
```

Comment:
Minor improvement: ext is now always a shared_ptr since this PR (See line 1275 in this file and line 1211 in the previous version), so this #ifdef could be removed (the #else part could be removed).